src/main/java/baubles/client/ClientProxy.java

```diff
@@ -38,17 +41,17 @@ public Object getClientGuiElement(int ID, EntityPlayer player, World world, int
 		}
 		return null;
 	}
-				
+
 	@Override
 	public World getClientWorld() {
 		return FMLClientHandler.instance().getClient().world;
 	}
-	
-	@Override
-	public void registerItemModels() {
-		ModelLoader.setCustomModelResourceLocation(Config.itemRing, 0, new ModelResourceLocation("baubles:Ring", "inventory"));
+
+	@SubscribeEvent
+	public static void registerItemModels(ModelRegistryEvent event) {
```

Comment:
In my opinion, events shouldnt be part of the proxy. Might want to move this either to the mod class or even its own class.